lib/plugins.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var _ = require('lodash');
+var prefix = 'gemini-';
+
+module.exports = {
+    load: function(gemini, options) {
+        var enabledPlugins = _.reject(_.pairs(options.plugins), function(p) {
```

Comment:
Not sure if this step is needed. If someone wants to disable a plugin the can just remove it from the config.